ansible/roles/bigbluebutton/molecule/default/prepare.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
This file is no longer referenced in `molecule.yaml` so it's possibly not needed.

Reply:
Please see comment: https://github.com/PrivacyLx/devops/pull/55/files#r598654577